src/Symfony/Component/WebLink/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/weblink",
```

Comment:
Maybe we could rename the namespace to `Weblink` instead to keep consistency between package name and namespace.